dev-docs/publisher-api-reference.md

```diff
@@ -2195,6 +2196,37 @@ If you don't see that message, you can assume the config object is valid.
 
 <hr class="full-rule" />
 
+<a name="module_pbjs.setBidderConfig"></a>
+
+### pbjs.setBidderConfig(options)
+
+This function is similar to [`setConfig`](#module_pbjs.setConfig), but is designed to support certain bidder-specific scenarios.
+
+Configuration provided through the [`setConfig`](#module_pbjs.setConfig) function is
+globally available to all bidder adapters. This makes sense because
+most of these settings are global in nature. However, there are use cases where different bidders require different data, or where certain parameters apply only to a given
+bidder. Use `setBidderConfig` when you need to support these cases.
+
+The page usage is:
+
+{% highlight js %}
+pbjs.setBidderConfig({
+   bidders: ["bidderA"],  // one or more bidders
+   config: {              // the bidder-specific config
+      bidderA: {
+         customArg: 'value'
+      }
+   }
+});
+{% endhighlight %}
+
+When 'bidderA' calls `getConfig('bidderA')`, it will receive the object that contains 'customArg'.
+If any other bidder calls `getConfig('bidderA')`, it will receive nothing.
+
+{: .alert.alert-info :}
+The `setBidderConfig` function will soon be used by the `schain` and
```

Comment:
Do we know when "soon" is? Will we remember to revise this note when that happens?

Reply:
A couple of weeks. Yes, I plan on remembering to change this note. Of course the memory isn't what it once was.

Reply:
:-) Just wanted to make sure. I've seen a lot of docs that haven't been updated in years that say "new" or "coming soon". Thanks!